### ISSUE TITLE
[Bug]: Search failed with shards_num over 128

### ISSUE THREAD

**Issue by NicoYuan1986** [milvus-io/milvus#19130]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:master-20220909-eabdc2b
- Deployment mode(standalone or cluster):standalone
- SDK version(e.g. pymilvus v2.0.0rc2):2.2.0.dev30
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Search failed with shards_num over 128.
However, the range of shards_num is [1,256] according to https://milvus.io/docs/v2.1.x/create_collection.md

### Expected Behavior

Modify the document or fix the error.

### Steps To Reproduce

```python
@pytest.mark.tags(CaseLabel.L2)
    @pytest.mark.parametrize("shards_num", [129])
    def test_search_with_non_default_shard_nums(self, auto_id, shards_num, _async):
        """
        target: test search with non_default shards_num
        method: connect milvus, create collection with several shard numbers , insert, load and search
        expected: search successfully with the non_default shards_num
        """
        self._connect()
        name = cf.gen_unique_str(prefix)
        collection_w = self.init_collection_wrap(name=name, shards_num=shards_num)
        dataframe = cf.gen_default_dataframe_data()
        collection_w.insert(dataframe)
        collection_w.load()
        vectors = [[random.random() for _ in range(default_dim)] for _ in range(default_nq)]
        collection_w.search(vectors[:default_nq], default_search_field,
                            default_search_params, default_limit,
                            default_search_exp, _async=_async,
                            check_task=CheckTasks.check_search_results,
                            check_items={"nq": default_nq,
                                         "limit": default_limit,
                                         "_async": _async})
```

### Milvus Log

```
[2022-09-09 14:31:39 - ERROR - pymilvus.decorators]: RPC error: [load_collection], <MilvusException: (code=1, message=, etcdserver: too many operations in txn request)>, <Time:{'RPC start': '2022-09-09 14:31:39.389518', 'RPC error': '2022-09-09 14:31:39.425758'}> (decorators.py:100)
[2022-09-09 14:31:39 - ERROR - ci_test]: Traceback (most recent call last):
  File "/Users/zilliz/nico/milvus/tests/python_client/utils/api_request.py", line 26, in inner_wrapper
    res = func(*args, **_kwargs)
  File "/Users/zilliz/nico/milvus/tests/python_client/utils/api_request.py", line 57, in api_request
    return func(*arg, **kwargs)
  File "/Users/zilliz/virtual-environment/test2/lib/python3.8/site-packages/pymilvus/orm/collection.py", line 474, in load
    conn.load_collection(self._name, replica_number=replica_number, timeout=timeout, **kwargs)
  File "/Users/zilliz/virtual-environment/test2/lib/python3.8/site-packages/pymilvus/decorators.py", line 101, in handler
    raise e
  File "/Users/zilliz/virtual-environment/test2/lib/python3.8/site-packages/pymilvus/decorators.py", line 97, in handler
    return func(*args, **kwargs)
  File "/Users/zilliz/virtual-environment/test2/lib/python3.8/site-packages/pymilvus/decorators.py", line 127, in handler
    ret = func(self, *args, **kwargs)
  File "/Users/zilliz/virtual-environment/test2/lib/python3.8/site-packages/pymilvus/decorators.py", line 79, in handler
    raise e
  File "/Users/zilliz/virtual-environment/test2/lib/python3.8/site-packages/pymilvus/decorators.py", line 50, in handler
    return func(self, *args, **kwargs)
  File "/Users/zilliz/virtual-environment/test2/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 661, in load_collection
    raise MilvusException(response.error_code, response.reason)
pymilvus.exceptions.MilvusException: <MilvusException: (code=1, message=, etcdserver: too many operations in txn request)>
 (api_request.py:39)
[2022-09-09 14:31:39 - ERROR - ci_test]: (api_response) : <MilvusException: (code=1, message=, etcdserver: too many operations in txn request)> (api_request.py:40)

```

**Comment by xiaofan-luan** [comment]
/assign @longjiquan 

we want to ensure shard number always < 64.
It doesn't make sense to create too many shard numbers. Also etcd txn limit is another consideration

**Comment by xiaofan-luan** [comment]
@NicoYuan1986 
Good catch, we won't support shard number >64, but the error message definitely need to improve

**Comment by longjiquan** [comment]
Will be fixed by #19776 
/unassign
/assign @NicoYuan1986

**Comment by NicoYuan1986** [comment]
The issue has been solved.
